### DISCUSSION TITLE
[求助] 更换代理方式后无法连接互联网

### DISCUSSION THREAD

**Discussion by miner233** [BeyondDimension/SteamTools#2599] [Q&A]
我的情况是这样的：
使用了 Watt Toolkit 的时候（当时是用 Hosts 代理模式的，没有问题），我用 FireFox 时就无法访问 GitHub，提示证书错误，然后我手贱试了试别的代理模式，也提示证书错误。
然后我下一次打开 Watt Toolkit 准备再次使用 Hosts 代理加速的，但是客户端出现了错误：
![image](https://user-images.githubusercontent.com/110521837/222943300-d018862e-4679-404f-938d-abb4d52ca1b0.png)
我发现 **除了 PAC 系统代理模式** 以外的 **所有代理方式** 均无法连接互联网，而且如果不挂 PAC 代理的话也无法连接至互联网。
![image](https://user-images.githubusercontent.com/110521837/222943079-4a2f4d6a-78e3-41f1-9f01-b940bbd419da.png)
版本：Windows 2.8.6 x64

**Comment by rmbadmin** [comment]
系统设置中关闭 网络代理
